music/src/music_vae/model.ts

```diff
@@ -1034,6 +1107,26 @@ class MusicVAE {
         encodedChordProgression;
   }
 
+  /**
+   * Encodes the input `Tensor`s into latent vectors.
+   *
+   * @param inputTensors A 3D `Tensor` of sequences to encode.
+   * @returns A `Tensor` containing the batch of latent vectors, sized
+   * `[inputTensors.shape[0], zSize]`.
+   */
+  async encodeTensors(inputTensors: tf.Tensor3D) {
```

Comment:
can we call these new methods from the old ones to avoid duplication and make it more readable?

Reply:
Talked offline, I don't want to support conditioning yet in the *Tensors methods.